### DISCUSSION TITLE
Have you considered moving the project to Ublue?

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [sodaliterocks/sodalite#94] [Discussion]
maybe moving the project there will make your life a little easier (or not)
https://github.com/ublue-os

**Comment by electricduck** [comment]
Sodalite is fairly low-maintenance for me these days. While I admire Ublue, moving Sodalite over to its framework would be more trouble than its worth.

**Reply by Heus-Sueh** [reply]
I see, I thought it would be easier to port the project there and you share almost the same idea, but I see that you have your own tools for this project.
